frontend/src/scenes/experiments/ExperimentView/DataCollectionCalculator.tsx

```diff
@@ -0,0 +1,267 @@
+                                        <div className="l4">
+                                            {humanFriendlyNumber(
+                                                trendCount +
+                                                    Math.ceil(trendCount * ((minimumDetectableChange || 5) / 100)) || 0
+                                            )}
+                                        </div>
+                                    </div>
+                                </>
+                            )}
+                            <div className="w-1/2">
+                                <div className="card-secondary">Recommended running time</div>
+                                <div>
+                                    <span className="l4">~{humanFriendlyNumber(trendExposure || 0)}</span> days
+                                </div>
+                            </div>
+                        </div>
+                    ) : (
+                        <div className="flex flex-wrap">
+                            {!experiment?.start_date && (
+                                <>
+                                    <div className="w-1/2">
+                                        <div className="card-secondary">Baseline Conversion Rate</div>
+                                        <div className="l4">{funnelConversionRate.toFixed(1)}%</div>
+                                    </div>
+                                    <div className="w-1/2">
+                                        <div className="card-secondary">Minimum Acceptable Conversion Rate</div>
+                                        <div className="l4">
+                                            {(funnelConversionRate + (minimumDetectableChange || 5)).toFixed(1)}%
+                                        </div>
+                                    </div>
+                                </>
+                            )}
+                            <div className="w-1/2">
+                                <div className="card-secondary">Recommended Sample Size</div>
+                                <div className="pb-4">
+                                    <span className="l4">~{humanFriendlyNumber(funnelSampleSize || 0)}</span> persons
+                                </div>
+                            </div>
+                            {!experiment?.start_date && (
+                                <div className="w-1/2">
+                                    <div className="card-secondary">Recommended running time</div>
+                                    <div>
+                                        <span className="l4">~{humanFriendlyNumber(runningTime || 0)}</span> days
+                                    </div>
+                                </div>
+                            )}
+                        </div>
+                    )}
+                    <div className="flex w-full mt-4">
```

Comment:
I'd say none of this data is necessary here anymore, all I want to see is the slider, and how it changes the recommended running time / sample size.

Reply:
<img width="1002" alt="image" src="https://github.com/PostHog/posthog/assets/7115141/13897c91-25ce-4232-a4d4-2fdf1b3d4d37">

Feels the rest is extra out of context